src/timers.rs

```diff
@@ -29,6 +29,8 @@ pub enum TriggerSource {
 }
 
 pub enum Prescaler {
```

Comment:
Isn't there already such an enum/variant in the PAC for that?
https://stm32-rs.github.io/stm32-rs/STM32H743x.html#TIM2:SMCR:ETPS

Reply:
Yes, but the PAC-specific value is not generic over all possible timer types (e.g. there's a separate, equivalent value for TIM1, etc.), so I've made it generic  here.

Reply:
Perhaps a PR should be made for the PAC?

Reply:
Ah. Hmm. I don't know what's easiest/best then.